compiler/p/codegen/OMRConstantDataSnippet.hpp

```diff
@@ -48,8 +49,7 @@ namespace OMR
 
 template <class T> class PPCConstant
    {
-   TR_Array<TR::Instruction *>   _instructionPairs;
-   int32_t                         _tocOffset;
```

Comment:
This is being done since pTOC handling will now be handled outside of `ConstantDataSnippet`. I've rewritten the relevant commit message to make this a bit clearer.